lib/ansible/plugins/inventory/script.py

```diff
@@ -29,6 +29,8 @@
         - The plugin does not cache results because external inventory scripts are responsible for their own caching.
         - To write your own inventory script see (R(Developing dynamic inventory,developing_inventory) from the documentation site.
         - To find the scripts that used to be part of the code release, go to U(https://github.com/ansible-community/contrib-scripts/).
+        - Since 2.19 using a directory as inventory source will no longer ignore .ini files by default,
+          but you can still update the configuration to do so.
```

Comment:
```suggestion
        - Since 2.19 using a directory as an inventory source will no longer ignore .ini files by default,
          but you can still update the configuration to do so.
```